src/toil/test/lib/dockerTest.py

```diff
@@ -199,28 +186,12 @@ def _testDockerPipeChainFn(job):
 
 def _testDockerPipeChainErrorFn(job):
     """
-    Return True if the command exit 1 | wc -l raises a CalledProcessError when run through 
+    Return True if the command exit 1 | wc -l raises a ContainerError when run through
     the docker interface
     """
     parameters = [ ['exit', '1'], ['wc', '-l'] ]
     try:
-        return dockerCheckOutput(job, tool='quay.io/ucsc_cgl/spooky_test', parameters=parameters)
-    except CalledProcessError:
+        dockerCheckOutput(job, tool='quay.io/ucsc_cgl/spooky_test', parameters=parameters)
+    except ContainerError:
         return True
     return False
-
-def _testDockerPermissions(job):
```

Comment:
Why is this test being removed?

Reply:
File permissions are already tested above (though perhaps it is a better idea to separate this from the orphan killing tests above).  I'll re-implement this when I attempt to rewrite the main dockerClean test as multiple subtests.